### PR TITLE
2.6.0 cleanup

### PR DESCRIPTION
**Pull request by btry**
Cleanup all unused globals in methods, dead code, and unused vars.

### CODE REVIEW

File: inc/form.class.php [hunk 1]
```diff
@@ -582,7 +578,6 @@ public function showFormList($rootCategory = 0, $keywords = '', $helpdeskHome =
       $table_target  = getTableForItemType('PluginFormcreatorTargets');
       $table_section = getTableForItemType('PluginFormcreatorSections');
       $table_question= getTableForItemType('PluginFormcreatorQuestions');
-      $where         = getEntitiesRestrictRequest( "", $table_form, "", "", true, false);
```
Comment (btry) [inline inc/form.class.php]: is never used

File: inc/form.class.php [hunk 2]
```diff
@@ -1170,10 +1161,6 @@ public  function getByQuestionId($questionId) {
     * @return Boolean true if success, false toherwize.
     */
    public function duplicate() {
-      global $DB;
-
-      $section             = new PluginFormcreatorSection();
-      $question            = new PluginFormcreatorQuestion();
```
Comment (btry) [inline inc/form.class.php]: are never used

File: inc/form.class.php [hunk 3]
```diff
@@ -1572,8 +1555,6 @@ public static function import($form = array()) {
       $entity   = new Entity;
       $form_cat = new PluginFormcreatorCategory;
 
-      $item = new self;
```
Comment (btry) [inline inc/form.class.php]: is never used

File: inc/form_validator.class.php [hunk 4]
```diff
@@ -20,11 +20,9 @@ class PluginFormcreatorForm_Validator extends CommonDBRelation {
    const VALIDATION_GROUP = 2;
 
    /**
-    * @see  CommondDBTM::prepareInputForAdd
```
Comment (orthagh) [inline inc/form_validator.class.php]: remove the whole dockblock

File: inc/question.class.php [hunk 5]
```diff
@@ -379,6 +361,7 @@ function warning_handler($errno, $errstr, $errfile, $errline, array $errcontext)
                   ldap_control_paged_result($ds, 1);
                   $sn            = ldap_search($ds, $config_ldap->fields['basedn'], $input['ldap_filter'], $attribute);
                   $entries       = ldap_get_entries($ds, $sn);
+                  unset($entries);
```
Comment (orthagh) [inline inc/question.class.php]: is unset needed ? it seems superflu to remove it

**Pull request merged**